docs/source/welcome/services.md

```diff
@@ -1,25 +1,42 @@
+:link: ../datamanagment/discovery_storage
+:link-type: doc
 
-We have many file storage solutions available to members of the Northeastern Community and provide tailored solutions to cater to both long-term high-capacity storage needs and dynamic storage requirements for active research. Our technical team works closely with researchers to design data management strategies, facilitating seamless data movement, backup, and restoration.
+We offer several storage solutions for active and archival use.
++++
+[Learn more »](../datamanagment/discovery_storage)
+:::
 
-### Classroom
+:::{grid-item-card} {octicon}`mortar-board;1.5em;sd-mr-1` Classroom
+:link: ../classroom/class_use
+:link-type: doc
 
-At Research Computing we extend our expertise beyond research contexts to provide comprehensive support for classroom use of NU's computational resources. Our skilled team collaborates with educators to integrate high-performance computing into curricula, enhancing students' learning experiences. We offer tailored workshops and training sessions that empower instructors to incorporate computational techniques into their courses. From introductory coding exercises to advanced simulations, we assist in designing educational content that leverages the power of our computing cluster. Our technical staff ensures that students have access to the necessary tools and resources, enabling them to explore real-world applications of computational methods.
+We support classroom use of HPC resources.
++++
+[Learn more »](../classroom/class_use)
+:::
+::::
 
 ## Consultations
 
-We encourage you to schedule a consultation with one of our staff members to receive personal, one-on-one assistance for your research computing and data storage needs. Consultations are available to any Northeastern student, faculty, or staff member. We can help you get started on the cluster, optimize your code, benchmark, install and use software packages, detail data storage options, and more.
+We encourage you to schedule a consultation with one of our staff members to receive personal, one-on-one assistance for your research computing, data storage, or classroom needs. Consultations are available to any Northeastern student, faculty, or staff member. We can help you get started on the cluster, optimize your code, benchmark, install and use software packages, detail data storage options, and more.
 
-We offer consultations by appointment most weekdays during regular business hours (9 a.m. to 5 p.m.). Just so you know, we follow the Northeastern University holiday schedule, so no consultations are available on holidays or during breaks. All consultations are conducted online through the Teams app.
+We offer consultations by appointment most weekdays during regular business hours (9 a.m. to 5 p.m.). We follow the Northeastern University holiday schedule, so no consultations are available on holidays or during breaks. All consultations are conducted online through the Teams app.
```

Comment:
Should it be application? Not sure if abbreviations should be used in documentation.